src/components/designer/link/Channel.tsx

```diff
@@ -42,6 +43,7 @@ const Channel: React.FC<Props> = ({ link, from, to }) => {
         />
       ),
     },
+    { label: l('isPrivate'), value: isPrivate ? 'true' : 'false' },
```

Comment:
I'm not sure if showing a `true` or `false` string here is the best. Maybe showing `Yes` or `No` would be better?

Reply:
Ok, I will leave it as is then.